app/common/data/migrations/versions/006_metadata_enum.py

```diff
@@ -0,0 +1,23 @@
+"""extends metadata enum
+
+Revision ID: 006_metadata_enum
+Revises: 005_collection_metadata
+Create Date: 2025-06-04 14:56:07.802724
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+revision = "006_metadata_enum"
+down_revision = "005_collection_metadata"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.execute(sa.text("ALTER TYPE metadata_event_key_enum ADD VALUE 'COLLECTION_SUBMITTED'"))
+
+
+def downgrade() -> None:
+    pass
```

Comment:
Hopefully we never need to/try to downgrade this 🤞

Reply:
Yeah what I've done here lines up with what Sarah/ Marc looked into here https://mhclgdigital.atlassian.net/wiki/spaces/FS/pages/554139677/Enums#Adding-values

I figured if we're going to squash migrations before prod copying and altering and destroying was probably more effort/ error prone than its worth